### PR TITLE
Fixes to support timescale 2.3.0

### PR DESCRIPTION
**Pull request by cevian**
The biggest change was to fix which error message triggers
decompression. That's the only functional change.

The rest are changes in the golden files to show that the
compression status has been fixed.

### CODE REVIEW

File: pkg/tests/end_to_end_tests/golden_files_test.go [hunk 1]
```diff
@@ -17,9 +17,10 @@ import (
 )
 
 var outputDifferWithoutTimescale = map[string]bool{"info_view": true}
-var outputDifferWithMultinode = map[string]bool{"views": true, "info_view": true, "support": true}
+var outputDifferWithMultinode = map[string]bool{"views": true, "info_view": true}
 var outputDifferWithExtension = map[string]bool{"support": true}
 var requiresTimescaleDB = map[string]bool{"views": true, "info_view": true, "support": true}
+var requiresSingleNode = map[string]bool{"support": true}
```
Comment (Harkishen-Singh) [inline pkg/tests/end_to_end_tests/golden_files_test.go:23]: Can we please group them into a single var declaration?

File: pkg/tests/testdata/expected/info_view-timescaledb-2-multinode.out [hunk 2]
```diff
@@ -55,8 +55,8 @@ SET
...
-  1 | cpu_usage   | cpu_usage  | 90 days          | t        | f        |                    24576 |                   32768 | {__name__,namespace,new_tag,node}   | 64 kB      |            65536 | -33.3333333333333300 |            1 |                 1
-  2 | cpu_total   | cpu_total  | 90 days          | t        | f        |                    24576 |                   32768 | {__name__,namespace,new_tag_2,node} | 64 kB      |            65536 | -33.3333333333333300 |            1 |                 1
+  1 | cpu_usage   | cpu_usage  | 90 days          | t        | t        |                    24576 |                   32768 | {__name__,namespace,new_tag,node}   | 64 kB      |            65536 | -33.3333333333333300 |            1 |                 1
+  2 | cpu_total   | cpu_total  | 90 days          | t        | t        |                    24576 |                   32768 | {__name__,namespace,new_tag_2,node} | 64 kB      |            65536 | -33.3333333333333300 |            1 |                 1
```
Comment (Harkishen-Singh) [inline pkg/tests/testdata/expected/info_view-timescaledb-2-multinode.out:59]: Can you say why is this change?

**Review by Harkishen-Singh — CHANGES_REQUESTED** [review]

**Review by antekresic — APPROVED** [review]

File: pkg/tests/testdata/expected/info_view-timescaledb-2-multinode.out (hunk 2 above)
Comment (cevian) [reply pkg/tests/testdata/expected/info_view-timescaledb-2-multinode.out:59]: Yes this was fixed in Timescaledb 2.3

File: pkg/tests/end_to_end_tests/golden_files_test.go (hunk 1 above)
Comment (cevian) [reply pkg/tests/end_to_end_tests/golden_files_test.go:23]: No, I don't like single var declarations in general

**Review by Harkishen-Singh — APPROVED** [review]

**Pull request merged**